components/AddPlantForm/index.js

```diff
@@ -0,0 +1,187 @@
+import styled from "styled-components";
+
+export default function AddPlantForm ({handleAddPlant}){
+
+    function handleSubmitAddPlant(event){
+        event.preventDefault();
+        const formData = new FormData(event.target);
+        const data = {}
+        formData.forEach((value, key) => {
+            if (key === "fertiliserSeason") {
+              if (!data[key]) {
+                data[key] = [];
+              }
+              data[key].push(value);
+            } else {
+              data[key] = value;
+            }
+          });
+        handleAddPlant(data)
+        event.target.reset();
+    }
+
+    return(
+        <StyledSection>
+            <StyledH2>Add plant</StyledH2>
+            <form onSubmit={handleSubmitAddPlant}>
+                <StyledFieldset>
+                    <label htmlFor="plantName">
+                    Plant name:
+                    </label>
+                    <StyledInput id="plantName" name="name" type="text" placeholder="e.g. Monstera" required></StyledInput>
+                </StyledFieldset>
+                <StyledFieldset>
+                    <label htmlFor="botanicalPlantName">
+                    Botanical plant name:
+                    </label>
+                    <StyledInput id="botanicalPlantName" name="botanicalName" type="text" placeholder="e.g. Monstera deliciosa" required></StyledInput>
+                </StyledFieldset>
+                <StyledFieldset>
+                    <label htmlFor="plantDescription">
+                    Description:
+                    </label>
+                    <StyledTextarea id="plantDescription" name="description" type="text" placeholder="e.g. Monstera deliciosa, also known as the Swiss cheese plant"></StyledTextarea>
+                </StyledFieldset>
+                {/*-----------------------------------------------------------*/}
+                <StyledFieldsetRadio>
+                    <legend> Light needs </legend>
+
+                    <StyledRadioInput id="light-full-shade" name="lightNeed" type="radio" value="Full Shade" required/>
+                    <StyledRadiolabel htmlFor="light-full-shade">Full Shade </StyledRadiolabel>
+
+                    <StyledRadioInput id="light-partial-shade" name="lightNeed" type="radio" value="Partial Shade" required/>
+                    <StyledRadiolabel htmlFor="light-partial-shade">Partial Shade </StyledRadiolabel>
+
+                    <StyledRadioInput id="light-full-sun" name="lightNeed" type="radio" value="Full Sun" required/>
+                    <StyledRadiolabel htmlFor="light-full-sun">Full Sun </StyledRadiolabel>
+                </StyledFieldsetRadio>
+                {/*-----------------------------------------------------------*/}
```

Comment:
wich meaning is this comment?

Reply:
It is just to visually separate sections of code within the form because it can be confusing to read and distinguish different sections